packages/notifi-react-card/lib/components/subscription/SubscriptionCardV1.tsx

```diff
@@ -79,14 +86,18 @@ export const SubscriptionCardV1: React.FC<SubscriptionCardV1Props> = ({
       break;
     case 'edit':
       view = (
-        <EditCardView
-          data={data}
-          classNames={classNames?.EditCard}
-          inputDisabled={inputDisabled}
-          inputLabels={inputLabels}
-          inputSeparators={inputSeparators}
-          allowedCountryCodes={allowedCountryCodes}
-        />
+        <>
+          <AlertListPreview eventTypes={data.eventTypes} />
```

Comment:
just curious -- why not just put it in editCardView?

if we expose `classNames`, it'd be nice to have this nested under EditCard

Reply:
oh, no, you're totally right. let me move it